README.md

```diff
@@ -17,8 +17,8 @@ The Azure Management Libraries for .NET is a higher-level, object-oriented API f
   <tr>
     <td>Compute</td>
     <td>Virtual machines and VM extensions<br>Virtual machine scale sets<br>Managed disks</td>
-    <td></td>
     <td valign="top">Azure container services<br>Azure container registry</td>
```

Comment:
"Azure container registry" has its own package and it's not part of compute. Should be either in its own raw or moved under "More services"